docs/quickstarts/ratify-cli.md

```diff
@@ -0,0 +1,119 @@
+# Ratify CLI
+
+This document provides a walkthrough on how to use the ratify command line to verify a sample image.
+
+## Verify notation signature with Ratify cli
+
+1. Download the latest Ratify CLI on a Linux AMD64 machine. Go to [release page](https://github.com/ratify-project/ratify/releases/) if you are on other platforms.
+```bash
+curl -L https://github.com/ratify-project/ratify/releases/download/v1.2.1/ratify_1.2.1_Linux_amd64.tar.gz | tar xvzC ~/bin/ ratify
+```
+
+2. Download a local verification certificate for sample image
+```bash
+curl -sSLO https://github.com/ratify-project/ratify/blob/v1.2.1/test/testdata/notation.crt
```

Comment:
seems we can update the URL to `https://[raw.githubusercontent.com/ratify-project/ratify/dev/test/testdata/notation.crt](https://raw.githubusercontent.com/ratify-project/ratify/dev/test/testdata/notation.crt)`